controllers/deploy.py

```diff
@@ -637,8 +637,6 @@ def postp(r, output):
 def email_channel():
     """
         RESTful CRUD controller for Inbound Email channels
-
-        @ToDo: Allow selection of a specific Channel for Alerts
```

Comment:
Yeah sorry for that, i was testing out some stuff (I might i have forgotten to change my branch to local), I'll revert them.